.github/CODEOWNERS

```diff
@@ -1,6 +1,6 @@
 # Regularly updated submodules
-/rocm-libraries/        @ROCm/TheRockInfra
-/rocm-systems/          @ROCm/TheRockInfra
+/rocm-libraries         @ROCm/TheRockInfra
+/rocm-systems           @ROCm/TheRockInfra
 
 # New patches are strongly discouraged. See patches/README.md.
 /patches/               @ScottTodd @marbre
```

Comment:
```suggestion
/patches                @ScottTodd @marbre
```
Do we need to adjust `patches` as well?

Reply:
Nope, that one is fine since git treats it as a directory. You can check by going to a file under that directory like https://github.com/ROCm/TheRock/blob/main/patches/README.md:

<img width="628" height="184" alt="image" src="https://github.com/user-attachments/assets/eafc4210-3a89-456d-bf8d-310f38a7230e" />

The submodules are treated more like "files" than "directories" for codeowners. See also what we did in IREE: https://github.com/iree-org/iree/blob/aba450e86f1cf8428347f58f630dada55dbf8f59/.github/CODEOWNERS#L14-L21